### PR TITLE
Pal 424 streaming local k8s end to end

### PR DESCRIPTION
**Pull request by dev930018**
Recommend building resource-service from PAL-867.

To ease testing, build and use the following Dockerfile (`docker build`) which contains lots of useful tools:
```
FROM alpine/k8s:1.14.9
VOLUME /tmp

ARG K8_VERSION=v1.15.3

COPY --from=golang:1.13-alpine /usr/local/go/ /usr/local/go/
ENV PATH="/usr/local/go/bin:${PATH}"

RUN apk add --update curl bash kafkacat nodejs npm

RUN go get -u github.com/hashrocket/ws && \
        mv /root/go/bin/ws /usr/bin/ws

RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/${K8_VERSION}/bin/linux/amd64/kubectl && \
      chmod +x ./kubectl && \
      mv ./kubectl /usr/local/bin/kubectl

LABEL service="k8stools" project="k8stools"
```

Example manual run-through before client is complete:
```
cd Palisade-examples
deployment/local-k8s/example-model/deployServicesToK8s.sh

kubectl run -it k8stools --image k8stools/k8stools:latest --image-pull-policy=Never --restart=Never ==rm -- bash

echo `curl -X POST palisade-service/api/registerDataRequest -H "content-type: application/json" --data \
'{
   "userId":"Alice",
   "resourceId":"file:/data/local-data-store/",
   "context":{
      "purpose":"SALARY"
   }
}'`

ws ws://filtered-resource-service/resource/$YOUR_TOKEN_HERE
> {"type":"PING","headers":{},"body":null}
> {"type":"CTS","headers":{},"body":null}
> {"type":"CTS","headers":{},"body":null}
> {"type":"CTS","headers":{},"body":null}
```

### CODE REVIEW

File: charts/deployment/templates/role.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Crown Copyright
```
Comment (kg981167224) [inline charts/deployment/templates/role.yaml:1]:
```suggestion
# Copyright 2021 Crown Copyright
```

File: charts/deployment/templates/service-account.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Crown Copyright
```
Comment (kg981167224) [inline charts/deployment/templates/service-account.yaml:1]:
```suggestion
# Copyright 2021 Crown Copyright
```

**Review by kg981167224 — DISMISSED** [review]
Minor copyright changes to new files

**Review by pd104923 — CHANGES_REQUESTED** [review]
1 minor thing about the licenses. Will try and run it

**Comment by dev930018** [comment]
Can now be run with:
```
deployment/local-k8s/example-runner/deployServicesToK8s.sh
deployment/local-k8s/example-runner/runFormattedK8sExample.sh
deployment/local-k8s/example-runner/verify.sh
```

File: pom.xml [hunk 3]
```diff
@@ -95,7 +95,7 @@
         <module>example-library</module>
         <module>example-runner</module>
         <module>performance</module>
-        <module>deploy-example-jars</module>
+        <module>deployment</module>
     </modules>
```
Comment (kg981167224) [inline pom.xml:99]: Does this change mean we can remove the repository for deploy-example-jars?

**Review by kg981167224 — APPROVED** [review]
Approved with 1 quick question.

File: pom.xml (hunk 3 above)
Comment (dev930018) [reply pom.xml:99]: Once merged, yes

**Review by kg981167224 — APPROVED** [review]

**Review by pd104923 — APPROVED** [review]

**Review by dev-958 — DISMISSED** [review]

**Review by dev-958 — APPROVED** [review]

**Pull request merged**